cms/djangoapps/contentstore/views/library.py

```diff
@@ -152,6 +152,14 @@ def _create_library(request):
         library = request.json.get('number', None)
         if library is None:
             library = request.json['library']
+        # Allow user to create the library only if they belong to the organization
+        if settings.FEATURES.get('RESTRICT_NON_ORG_COURSE_CREATION'):
```

Comment:
@farhaanbukhsh, would it make sense to check and restrict access based on this only when the `ENABLE_CREATOR_GROUP` feature flag is set enabled?